### PR TITLE
Edit protocol basics to improve readability

### PR DESCRIPTION
**Pull request by s-n-park**
Edited wording a bit to try and improve readability

Flesch-Kincaid Score previous: 35.39
After re-work: 50.2

### CODE REVIEW

File: docs/1.concepts/basics/protocol.md [hunk 1]
```diff
@@ -4,42 +4,48 @@ title: What is NEAR?
...
 
 ### 🧑‍💻 Great Developer Experience
-1. Developers can build smart contracts using **Javascript** or **Rust**
-2. **Simple onboarding**, thanks to its complete documentation and a multitude of examples
-3. NEAR DevRel holds **office hours every weekday**, where anybody can participate
-4. 30% of gas fees goes to the developers
+1. Build smart contracts with **Javascript** or **Rust**
+2. **Simple onboarding**, thanks to its complete documentation and examples
+3. Get answers and learn at NEAR DevRel **office hours**, where anybody can participate
+4. Earn 30% of gas fees
```
Comment (gagdiez) [inline docs/1.concepts/basics/protocol.md:46]:
```suggestion
4. Earn from your contract's gas fees
```

Comment (gagdiez) [reply docs/1.concepts/basics/protocol.md:46]: I see our original version is a bit misleading

**Review by gagdiez — CHANGES_REQUESTED** [review]

File: docs/1.concepts/basics/protocol.md [hunk 2]
```diff
@@ -4,42 +4,48 @@ title: What is NEAR?
...
-NEAR is a technical marvel, offering built-in features such as named accounts and account abstraction. For developers, NEAR offers all the infra-structure needed for their applications, from smart contracts to indexers. All while being interoperable with other chains.
+NEAR is a technical marvel, offering built-in features such as named accounts and account abstraction. For developers, NEAR offers everything needed for their applications, from smart contracts to indexers. All while being interoperable with other chains.
 
 ### ⭐ Simple to Use 
-1. Accounts are easy to remember and share thanks to built-in [**Named accounts**](../protocol/account-model.md) (as `alice.near`)
-2. Simple sign-up: accounts can be created through [email](https://near.org/signup) or [telegram](https://web.telegram.org/k/#@herewalletbot)
-3. Transactions are **incredibly fast** _(~1.3s transactions)_ and **cheap** _(< 1¢ in fees)_
-4. No need to acquire crypto thanks to **built-in account abstraction**
-5. Enhanced usability and security thanks to a rich [Access Keys](../protocol/access-keys.md) system
+1. [**Named accounts**](../protocol/account-model.md) let you use easy names like `alice.near`.
```
[the remark this answers is not in the document]
Comment (s-n-park) [reply docs/1.concepts/basics/protocol.md]: I revised to:
> [**Named accounts**](../protocol/account-model.md) let you use easily remember and share names like `alice.near` for your blockchain addresses.

But lmk if you think the original is still clearer happy to change it back:
> Accounts are easy to remember and share thanks to built-in [**Named accounts**](../protocol/account-model.md) (as `alice.near`)

File: docs/1.concepts/basics/protocol.md [hunk 3]
```diff
@@ -4,42 +4,53 @@ title: What is NEAR?
...
+NEAR is a technical marvel, offering built-in features such as named accounts and account abstraction. For developers, NEAR offers everything needed for their applications, from smart contracts to indexers. All while being interoperable with other chains.
 
 ### ⭐ Simple to Use 
-1. Accounts are easy to remember and share thanks to built-in [**Named accounts**](../protocol/account-model.md) (as `alice.near`)
-2. Simple sign-up: accounts can be created through [email](https://near.org/signup) or [telegram](https://web.telegram.org/k/#@herewalletbot)
-3. Transactions are **incredibly fast** _(~1.3s transactions)_ and **cheap** _(< 1¢ in fees)_
-4. No need to acquire crypto thanks to **built-in account abstraction**
-5. Enhanced usability and security thanks to a rich [Access Keys](../protocol/access-keys.md) system
+1. [**Named accounts**](../protocol/account-model.md) let you use easily remember and share names like `alice.near` for your blockchain addresses.
```
Comment (gagdiez) [inline docs/1.concepts/basics/protocol.md:32]:
```suggestion
1. Use [**named accounts**](../protocol/account-model.md) like `alice.near`
```

**Review by gagdiez — APPROVED** [review]

**Pull request merged**